eZ/Bundle/EzPublishCoreBundle/Command/DebugConfigResolverCommand.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class DebugConfigResolverCommand extends ContainerAwareCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function configure()
+    {
+        $this->setName('ezplatform:debug:config-resolver');
+        $this->setAliases(['ezplatform:debug:config']);
+        $this->setDescription('Debug / Retrive parameter from Config Resolver');
+        $this->addArgument(
+            'parameter',
+            InputArgument::REQUIRED,
+            'The configuration resolver parameter to return, for instance "languages" or "http_cache.purge_servers"'
+        );
+        $this->addOption(
+            'oneline',
+            'o',
+            InputOption::VALUE_NONE,
+            'Only return value, for automation / testing use on a single line.',
+        );
+        $this->setHelp(<<<EOM
+Outputs a given config resolver parameter, more commonly known as a SiteAccess setting.
```

Comment:
IMHO there is no reason for limiting this command to the siteaccess scope only. In the long term having options `--namespace` and `--scope` will be even more useful 😉

Reply:
So you are saying by default we let scope be set by siteaccess, burt add optional arguments for namespace and scope then, right?